apps/ui/src/views/Proposal/Votes.vue

```diff
@@ -116,7 +116,7 @@ watch([sortBy, choiceFilter], () => {
     class="bg-skin-bg sticky top-[112px] lg:top-[113px] z-40 border-b flex space-x-1 font-medium"
   >
     <div class="ml-4 max-w-[218px] w-[218px] truncate">Voter</div>
-    <div class="grow w-[40%] truncate">
+    <div class="grow w-[40%]">
```

Comment:
That fixes the issue, but makes truncate no longer work. Do you think there is something we can do about it without too much work (without rewriting our dropdowns to use poppler/portals)?

If not I think we can live without making this column truncatable.

Reply:
Did not find anything viable to make this works purely with css.

Will try this one https://github.com/tailwindlabs/headlessui/issues/30#issuecomment-1290548498

Reply:
Does not seems like an easy fix. I propose to merge without the truncate (will have uneven column width on very small screen), and open a new issue for this refactoring